backend/simulation.py

```
@@ -0,0 +1,330 @@
+import random
+import numpy as np
+import pandas as pd
```

Ruff (0.11.9)

`pandas` imported but unused

Remove unused import: `pandas`

(F401)